src/main/groovy/org/pih/warehouse/product/ProductSupplierFilterCommand.groovy

```diff
@@ -23,6 +23,18 @@ class ProductSupplierFilterCommand extends PaginationCommand implements Validate
 
     String order
 
+    String format
+
+    Boolean exportResults
+
+    @Override
+    Integer getMax() {
+        if (exportResults) {
+            return Integer.MAX_VALUE
```

Comment:
I have mixed feelings about that. What would happen if the offset was `null`? wouldn't it return all?